extensions/adobe/experience/adcloud/addeliverydetails.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/adDeliveryDetails",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Ad Delivery Details",
+  "type": "object",
+  "description": "AdCloud Search Ad Delivery Details  ",
```

Comment:
WE need to use correctly branded names AdCloud -> "Adobe Advertising Cloud", this will be fixed in the other schemas shortly too.

Reply:
My commit resolved this